x-pack/plugins/canvas/public/expression_types/arg_types/container_style/index.js

```diff
@@ -24,11 +25,19 @@ const wrap = Component =>
     },
   })(Component);
 
-export const containerStyle = () => ({
+const containerStyleUI = intl => ({
   name: 'containerStyle',
-  displayName: 'Container Style',
-  help: 'Tweak the appearance of the element container',
+  displayName: intl.formatMessage({
+    id: 'xpack.canvas.expressionTypes.containerStyle.containerStyleLabel',
+    defaultMessage: 'Container Style',
+  }),
+  help: intl.formatMessage({
+    id: 'xpack.canvas.expressionTypes.containerStyle.containerStyleHelpText',
+    defaultMessage: 'Tweak the appearance of the element container',
+  }),
   default: '{containerStyle}',
   simpleTemplate: templateFromReactComponent(wrap(SimpleTemplate)),
   template: templateFromReactComponent(wrap(ExtendedTemplate)),
 });
+
+export const containerStyle = injectI18n(containerStyleUI);
```

Comment:
As noted in the the common functions pull, this approach is not appropriate. We shouldn't be requiring our external plugin authors to interface with the internal `intl` service. We should do this transparently. 

I suspect we'll need a sync here, as this approach is way outside of what was discussed.